app/database/models/member.py

```diff
@@ -49,11 +54,19 @@ def get_member_by_labmem_number(cls, gid: int, lab_member_number: int) -> "Membe
             None: If the member doesn't exist.
         """
 
-        if not gid:
-            raise ValueError("gid or uid must be none-zero.")
+        if not gid or not lab_member_number:
```

Comment:
This was correct before as "if not gid or not uid:". The uid is the primary key (paired with the gid) in the database. Without it, the entry can't exist. As well, the uid can never change, whilst the lab member number can at any point, so it would not be good as it is not static.

Reply:
Yep, gid or uid should be the only to identify a member

Reply:
My bad! The github preview made the context look like it was for get_member.